### ISSUE TITLE
Enhancement: Seminars not listed in past seminars list

### ISSUE THREAD

**Issue by reikdas** [PurduePL/purduepl.github.io#200]
### State the issue you want to address

Seminars 2024-8-23 and 2024-8-30 don't show up in the past seminars list.

**Comment by Pat-Lafon** [comment]
Yeah, the logic for this looks rather hairy/unnecessary? https://github.com/PurduePL/purduepl.github.io/blob/12624a33e6e497fcb40055335844a31930f5d124/seminars.html#L50-L59
